src/main/java/ognl/OgnlRuntime.java

```diff
@@ -2767,12 +2767,15 @@ public static Method getGetMethod(OgnlContext context, Class targetClass, String
     {
         // Cache is a map in two levels, so we provide two keys (see comments in ClassPropertyMethodCache below)
         Method method = cacheGetMethod.get(targetClass, propertyName);
+        if (method == ClassPropertyMethodCache.NULL_REPLACEMENT) {
+            return null;
+        }
         if (method != null)
             return method;
 
         // By checking key existence now and not before calling 'get', we will save a map resolution 90% of the times
-        if (cacheGetMethod.containsKey(targetClass, propertyName))
-            return null;
+//        if (cacheGetMethod.containsKey(targetClass, propertyName))
+//            return null;
```

Comment:
Please remove these line instead of commenting them out. Also remove the comment above.

Reply:
done